health/micro-ui/web/packages/modules/campaign-manager/src/components/AppPreview.js

```
@@ -1,9 +1,105 @@
+            backgroundColor: activeTab === tab.productVariantId ? "#C84C0E" : "transparent",
+            color: activeTab === tab.productVariantId ? "#fff" : "#0B0C0C",
+            cursor: "pointer",
+            fontSize: "14px",
+            fontWeight: activeTab === tab.productVariantId ? "600" : "400",
+            borderRadius: "4px 4px 0 0",
+            transition: "all 0.2s ease",
+            whiteSpace: "nowrap",
+            minWidth: "fit-content"
+          }}
+          onClick={() => onTabChange(tab.productVariantId)}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)